src/forecast/prediction.py

```diff
@@ -33,14 +23,23 @@ def forecast(self):
         return fcst
 
 
-if __name__ == '__main__':
-    print("done!")
-
-
-
-
+class FcstAndOutputs:
+    def __init__(self, df_fcst_encode: pd.DataFrame, data_path_parent_level: int = 2, selected_model=None):
+        if selected_model is None:
```

Comment:
Google's practice is to just use " if not selected_model"

Reply:
updated!